controllers/apply/awards-for-all/fields.js

```diff
@@ -867,6 +955,13 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                         en: 'Enter your organisation’s charity number',
                         cy: ''
                     })
+                },
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `Organisation’s charity number must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
`Charity registration number must be ${FREE_TEXT_MAXLENGTH.large} characters or less`